hack/generate-proto.sh

```diff
@@ -27,9 +27,7 @@ if [ "`command -v protoc-gen-grpc-gateway`" = "" ]; then
 fi
 
 if [ "`command -v goimports`" = "" ]; then
-  export GO111MODULE="off"
-  go get golang.org/x/tools/cmd/goimports
-  export GO111MODULE="on"
+  go install -mod=vendor ./vendor/golang.org/x/tools/cmd/goimports
```

Comment:
why change this?

Reply:
I already had `GOPATH/bin` in my `PATH`. In my case, I'm running `make codegen` for the first time on this laptop, and I didn't had `goimport` installed.
The problem seem to be a change in go get in 1.22 https://go.dev/doc/go1.22#go-command.
> go get is no longer supported outside of a module in the legacy GOPATH mode (that is, with GO111MODULE=off). Other build commands, such as go build and go test, will continue to work indefinitely for legacy GOPATH programs.

Also, the current change makes the installation method same as that of other Go tools in this project.